src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -37,6 +37,21 @@ mocha.describe('readings API', () => {
 				});
 
 				// Add C2 here
```

Comment:
Small item: Can you please remove the comment about adding C2 since the following code does this.